Formula/gtk+.rb

```diff
@@ -1,22 +1,19 @@
 class Gtkx < Formula
   desc "GUI toolkit"
   homepage "https://gtk.org/"
+  url "https://download.gnome.org/sources/gtk+/2.24/gtk+-2.24.33.tar.xz"
+  sha256 "ac2ac757f5942d318a311a54b0c80b5ef295f299c2a73c632f6bfb1ff49cc6da"
   license "LGPL-2.0-or-later"
+  revision 1
```

Comment:
As mentioned in extended commit message, I revision bumped `gtk+` to avoid potential conflict for users who installed `gtk+` and `gtk+3` (relatively common situation as we migrate `gtk+` formulae to `gtk+3`)

I didn't revision bump `gtk+3` as it shouldn't impact most users. It may cause minor issue for build-from-source users who install a dependent that may need `gtk-update-icon-cache` (e.g. `evince`) but I think this doesn't meet our current revision bump requirements (broken for majority of users or security issue).